DistributedCollective/Sovryn-frontend
# Bug/1204 language detection

Resolves #1204

- Fix language code format of pt-BR.
- Prevent conflict with i18next of \@sovryn/react-wallet.
- Correct i18next and i18next-browser-languagedetection configurations
- use constants for Language Toggle

Steps to test:
- Ensure that localStorage is clear for the Sovryn dapp link you are using - specifically the `i18nextLng` and `i18nextLng_dapp` keys should not be present.
- Set default browser language to "Portuguese (Brazil)" (or another supported language, but this one has most translations present in dapp)
- Load the Sovryn dapp link
- The dapp should be shown with the default browser language already set if it is supported (or revert to English if an unsupported language)
